src/main/kotlin/no/skatteetaten/aurora/mokey/CacheWarmup.kt

```diff
@@ -0,0 +1,14 @@
+package no.skatteetaten.aurora.mokey
+
+import no.skatteetaten.aurora.mokey.model.Environment
+import no.skatteetaten.aurora.mokey.service.ApplicationDataServiceCacheDecorator
+import org.springframework.beans.factory.InitializingBean
+import org.springframework.stereotype.Component
+
+@Component
+class CacheWarmup(val applicationDataService: ApplicationDataServiceCacheDecorator) : InitializingBean {
```

Comment:
Akkurat denne koden liker faktisk jeg at ligger i Main slik at man vet at det er en initilizing Bean når man starter prosjektet.

Reply:
Ulempen med å ha for mye i main er at det påvirker auto-oppsettet av tester når man bruker "slices" (som f.eks bare webtester). Da vil man kanskje ikke ha en ApplicationDataServiceCacheDecorator i applikasjonscontexten, og da smeller det. Blir mer manuelt arbeid i testene da.

Reply:
Legg det i en egen klasse i men legg denne i Main fila? Jeg synes det er fint at oppstartslogikk er i 1 fil.